internal/docker/docker.go

```diff
@@ -118,21 +120,26 @@ type GenerateRequest struct {
 // ReleaseRequest contains all the information required for a language
 // container to run the release command.
 type ReleaseRequest struct {
-	// cfg is a pointer to the [config.Config] struct, holding general configuration
+	// Cfg is a pointer to the [config.Config] struct, holding general configuration
 	// values parsed from flags or environment variables.
 	Cfg *config.Config
-	// state is a pointer to the [config.LibrarianState] struct, representing
+	// GlobalConfig is a pointer to the [config.GlobalConfig] struct, holding
+	// global files configuration in a language repository.
+	GlobalConfig *config.GlobalConfig
+	// State is a pointer to the [config.LibrarianState] struct, representing
 	// the overall state of the generation and release pipeline.
 	State *config.LibrarianState
-	// libraryID specifies the ID of the library to release.
+	// LibraryID specifies the ID of the library to release.
 	LibraryID string
-	// libraryID specifies the version of the library to release.
+	// LibraryVersion specifies the version of the library to release.
 	LibraryVersion string
-	// output specifies the empty output directory into which the command should
-	// generate code
+	// Output specifies the empty output directory into which the command should
+	// generate code.
 	Output string
-	// RepoDir is the local root directory of the language repository.
-	RepoDir string
+	// PartialRepoDir is the local root directory of language repository contains
+	// files that make up libraries and global files.
+	// This is the directory that container can access.
+	PartialRepoDir string
```

Comment:
How are you imagining this field being set? I am wondering if this should be private to this package. I don't think anything outside of this package needs access to this, right?